egs/wsj/s5/steps/cleanup/analyze_train_lats.sh

```diff
@@ -0,0 +1,111 @@
+#! /bin/bash
+
+# Copyright 2016  Vimal Manohar
+# Apache 2.0
+
```

Comment:
there needs to be at least a few sentences here explaining what this script is for, and also a more informative usage message.